streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/application/impl/SparkApplicationActionServiceImpl.java

```diff
@@ -508,8 +444,7 @@ private Tuple2<String, String> getUserJarAndAppConf(
                 FlinkSql flinkSql = flinkSqlService.getEffective(application.getId(), false);
                 AssertUtils.notNull(flinkSql);
```

Comment:
Does sparksql need to be distinguished from FlinkSql and stored in a different table

Reply:
![image](https://github.com/user-attachments/assets/7f521e07-7403-4735-9507-449fec23018e)
Does it need to be modified here